README.md

```diff
@@ -34,30 +34,44 @@ create -> launch
 learn  <-  measure
 ```
 
-We want to rapidly iterate our research-first mindset by constantly launching with an MVP implementations for every initiative.  
+#### Practically speaking though?
+
+Pop this in your terminal. This is your jumping off point. This is the only thing you need to do to get started.
+
+That is also not entirely true. This whole thing will always be a work-in-progress. When you run the command below, if there's something that breaks, or something you don't understand, or something is missing... awesome. Add to the project.
+
+We're never done.
```

Comment:
Proposed changes while keeping with the spirit.

```
Run the commands below in your terminal.

[commands here]

That will configure everything for you to get started. If something breaks, is missing, or is confusing...awesome. Open a pull request and let's add it to the project.

We're never done.
```

And we can keep the "We want to rapidly iterate..." in the `### Process` section.

Reply:
done

Reply:
Not sure why but it looks like the original changes are back? Maybe from my `gt sync`.